### PR TITLE
Certonly [needs revisions]

### PR DESCRIPTION
**Pull request by pde**
Merging #1577 with some extra clarification, and a logic change :)

### CODE REVIEW

File: letsencrypt/cli.py [hunk 1]
```diff
@@ -370,14 +370,16 @@ def diagnose_configurator_problem(cfg_type, requested, plugins):
...
                    .format(requested, plugins[requested].problem))
     elif cfg_type == "installer":
-        if os.path.exists("/etc/debian_version"):
-            # Debian... installers are at least possible
-            msg = ('No installers seem to be present and working on your system; '
-                   'fix that or try running letsencrypt with the "certonly" command')
+        if os.path.exists("/etc/debian_version") and le_util.exe_exists("apachectl"):
+            # Debian + Apache... installers are at least possible
+            msg = ('No certificate installer plugins were found on your system; '
+                   'fix that or run "letsencrypt certonly" to get a certificate '
```
Comment (bmw) [inline letsencrypt/cli.py]: nit: Perhaps the wording should be something like `'fix that or run Let's Encrypt with "certonly" on the command line'`.

Reason for this is i doubt many people will be running the client without `letsencrypt-auto` initially, but I don't like baking `letsencrypt-auto` into the command either.

File: letsencrypt/cli.py [hunk 2]
```diff
@@ -370,14 +370,16 @@ def diagnose_configurator_problem(cfg_type, requested, plugins):
...
+                   'you can install manually.')
         else:
             # XXX update this logic as we make progress on #788 and nginx support
-            msg = ('No installers are available on your OS yet; try running '
-                   '"letsencrypt-auto certonly" to get a cert you can install manually')
+            msg = ('No certificate installer plugins are available for your OS/'
+                   'webserver yet; try running "letsencrypt certonly" to get a '
```
Comment (bmw) [inline letsencrypt/cli.py]: Same as above.

**Comment by Sveder** [comment]
Awesome job!
I think this means you can mark https://github.com/letsencrypt/letsencrypt/issues/1576 as done :)

**Comment by bmw** [comment]
There have been suggested revisions on this for 6 months. With this in mind, I'm closing this PR to try and clean up our PR queue.

If you open a new PR with revisions, I'll happily review.

**Pull request closed without merging**